lib/ex_unit/lib/ex_unit/formatter.ex

```diff
@@ -65,17 +65,20 @@ defmodule ExUnit.Formatter do
       "Finished in 0.2 seconds (0.2s on load, 0.01s on tests)"
 
   """
-  @spec format_time(run_us, load_us) :: String.t
+  @spec format_time(run_us, load_us) :: String.t()
   def format_time(run_us, nil) do
     "Finished in #{run_us |> normalize_us |> format_us} seconds"
   end
 
   def format_time(run_us, load_us) do
-    run_us  = run_us |> normalize_us
+    run_us = run_us |> normalize_us
     load_us = load_us |> normalize_us
 
     total_us = run_us + load_us
-    "Finished in #{format_us total_us} seconds (#{format_us load_us}s on load, #{format_us run_us}s on tests)"
+
+    "Finished in #{format_us(total_us)} seconds (#{format_us(load_us)}s on load, #{
+      format_us(run_us)
+    }s on tests)"
```

Comment:
Please split this message into more than one string and concatenate through `<>` so that we don't get the ugly split in `#{`. This is described in example 4 (which I just added) of #6643.